apps/api-gateway/src/issuance/issuance.service.ts

```diff
@@ -162,5 +162,12 @@ export class IssuanceService extends BaseService {
         const payload = { orgId, userDetails };
         return this.sendNatsMessage(this.issuanceProxy, 'delete-issuance-records', payload);
     }
+    async getFileDetailsAndFileDataByFileId(orgId: string, fileId: string): Promise<{ response: object }> {
+        const payload = {
+          orgId,
+          fileId
+        };
+        return this.sendNats(this.issuanceProxy, 'issued-file-data-and-file-details', payload);
```

Comment:
Recently, we've been using the `sendNatsMessage` function. In this function, there's no need to add response as a key, you can use Promise<object> instead. Please refer to the existing functions for reference.

Reply:
resolved